eZ/Bundle/EzPublishCoreBundle/Command/NormalizeImagesPathsCommand.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use eZ\Publish\Core\FieldType\Image\ImageStorage\Gateway\DoctrineStorage;
+use eZ\Publish\Core\IO\FilePathNormalizerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class NormalizeImagesPathsCommand extends Command
+{
+    const BEFORE_RUNNING_HINTS = <<<EOT
+<error>Before you continue:</error>
+- Make sure to back up your database.
+- Run this command in production environment using <info>--env=prod</info>
+- Manually clear SPI/HTTP cache after running this command.
+EOT;
+
+    /** @var \eZ\Publish\Core\FieldType\Image\ImageStorage\Gateway\DoctrineStorage */
+    private $imageGateway;
+
+    /** @var \eZ\Publish\Core\IO\FilePathNormalizerInterface */
+    private $filePathNormalizer;
+
+    public function __construct(
+        DoctrineStorage $imageGateway,
+        FilePathNormalizerInterface $filePathNormalizer
+    ) {
+        parent::__construct();
+
+        $this->imageGateway = $imageGateway;
+        $this->filePathNormalizer = $filePathNormalizer;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('ezplatform:normalize-image-paths')
```

Comment:
Following existing convention (`ibexa:images:resize-original`):
```suggestion
            ->setName('ezplatform:images:normalize-paths')
```

Make sure to call it `ibexa:images:normalize-paths` on a merge up